Packages/com.unity.inputsystem/InputSystem/Utilities/Touch.cs

```diff
@@ -3,6 +3,12 @@
 
 namespace ISX
 {
+    public enum TouchType
+    {
+        Direct,
+        Indirect,
+        Stylus
+    }
```

Comment:
This one... IDK :) I have no idea what a "direct" touch and an "indirect" touch is. Seems very iOS-specific. Maybe it's just a naming thing. "Stylus touch" intuitively seems like it should go under pen support, but there's probably a good point to be made for discerning those kind of contacts on mobiles where pen support isn't as explicit as on, say, a Surface laptop.

Reply:
Direct touches are those usual touches performed via touchscreen.

An example of indirect touch is for ex from Siri remote, you click on touchpad and it's reported as indirect touch - https://developer.apple.com/documentation/uikit/uitouchtype/1618107-indirect

Mantas said that indirect touch also exist on Android, but haven't seen that yet.